tests/acceptance/features/apiLocks/lockFiles.feature

```diff
@@ -214,18 +214,22 @@ Feature: lock files
       | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario: viewer cannot lock a file in the shares using file-id
+  Scenario Outline: viewer cannot lock a file in the shares using file-id
```

Comment:
```suggestion
  Scenario Outline: resource shared with other role except editor cannot lock a file in the shares using file-id
```

Reply:
I think keeping "viewer" in the scenario gives more clearance. May be we can make it like:
`viewer with permissions other than editor cannot lock a file in the shares using file-id`

Reply:
`Scenario Outline: viewer cannot lock a file in the shares using file-id` is also fine